analyzers/tests/SonarAnalyzer.UnitTest/Rules/CertificateValidationCheckTest.cs

```diff
@@ -34,37 +31,38 @@ namespace SonarAnalyzer.UnitTest.Rules
     [TestClass]
     public class CertificateValidationCheckTest
     {
+        private readonly VerifierBuilder builderCS = new VerifierBuilder<CS.CertificateValidationCheck>()
```

Comment:
couldn't you use here GetAdditionalReferences?

it's not a huge duplication but less is more...

Reply:
A clean approach would be like this, but it produces more lines than it saves in this specific case.
```
private readonly VerifierBuilder builderCS = WithReferences<CS.CertificateValidationCheck>();
private readonly VerifierBuilder builderVB = WithReferences<VB.CertificateValidationCheck>();

        private static VerifierBuilder WithReferences<TAnalyzer>() where TAnalyzer : DiagnosticAnalyzer, new() =>
            new VerifierBuilder<TAnalyzer>()
                .AddReferences(MetadataReferenceFacade.SystemNetHttp)
                .AddReferences(MetadataReferenceFacade.SystemSecurityCryptography)
                .AddReferences(NetStandardMetadataReference.Netstandard);
```
I gave it a try with a static instance to duplicate it. The original `GetAdditionalReferences` approach should be preferably removed on the way.